### DISCUSSION TITLE
Files not storing in stacks folder

### DISCUSSION THREAD

**Discussion by samuellhwarren** [louislam/dockge#660] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I currently have 14 stacks, but only 2 managed to get their files created in my stacks folder.

### 🐻 Dockge Version

1

### 💻 Operating System and Arch

aarch64 

### 🌐 Browser

Chrome

### 🐋 Docker Version

Docker version 27.3.1, build ce12230

**Comment by mizady** [comment]
We will need more info to determine a cause or at least next steps.  the compose file with at least the volumes section of the service(s) would be a good start.  Just remember to clean any passwords or sensitive data before posting.

**Comment by samuellhwarren** [comment]
I forgot to close this. I tried changing a couple of locations and it worked.
Sam Warren
253.797.4789
iPhone.iTypos.iApologize
Samuellhwarren.com

On Nov 18, 2024, at 5:37 PM, Lance Cain ***@***.***> wrote:

﻿

We will need more info to determine a cause or at least next steps. the compose file with at least the volumes section of the service(s) would be a good start. Just remember to clean any passwords or sensitive data before posting.
